### PR TITLE
Workaround for AKS and Istio compatibility

### PR DESCRIPTION
**Pull request by ffais**
### Workaround for AKS and Istio compatibility

Temporary workaround for these issues:

- #84

### CODE REVIEW

File: pkg/service/sniffer/runtime/docker.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ func (d DockerBridge) BuildTcpdumpCommand(containerId *string, netInterface stri
 	d.tcpdumpContainerName = "ksniff-container-" + utils.GenerateRandomString(8)
 	containerNameFlag := fmt.Sprintf("--name=%s", d.tcpdumpContainerName)
 
-	command := []string{"docker", "--host", "unix:///host/var/run/docker.sock",
+	command := []string{"docker", "--host", "unix:///host/run/docker.sock",
```
Comment (bostrt) [inline pkg/service/sniffer/runtime/docker.go:32]: Would you mind removing this change for now? I think we're still trying to decide the best solution in #82

Comment (ffais) [reply pkg/service/sniffer/runtime/docker.go]: Of course, no problem.

**Comment by bostrt** [comment]
Hello! Apologies on the late reply. I have permission to merge commits now so I can help out some more. 

This PR is nearly ready. I just ask that you create a new string variable containing `"ksniff-privileged"` and use it in the changes for this PR and also here https://github.com/eldadru/ksniff/blob/master/kube/kubernetes_api_service.go#L138

**Comment by ffais** [comment]
Hi @bostrt,
I'm using "**privilegedContainerName**" to store the container name, let me know if it's correct.

**Comment by bostrt** [comment]
Thanks! I'll check how this is going to conflict with some other changes coming related to privileged containers.

**Comment by bostrt** [comment]
Hey @ffais Some big changes were merged recently and now your PR is conflicting. Would you mind addressing those and updating your PR?

**Comment by ffais** [comment]
Hi @bostrt, all conflicts have been resolved.

**Comment by bostrt** [comment]
Many thanks @ffais!

**Pull request merged**